site/content/docs/v1.12/restore-resource-modifiers.md

```diff
@@ -80,7 +80,7 @@ resourceModifierRules:
 version: v1
 resourceModifierRules:
 - conditions:
-    groupKind: deployments.apps
+    groupResource: deployments.apps
```

Comment:
can you add docs for label selector example.

Reply:
added